libexec/rc/rc.conf

```diff
@@ -135,8 +136,8 @@ wpa_supplicant_flags="-s"	# Extra flags to pass to wpa_supplicant
 wpa_supplicant_conf_file="/etc/wpa_supplicant.conf"
 #
 firewall_enable="NO"		# Set to YES to enable firewall functionality
-firewall_script="/etc/rc.firewall" # Which script to run to set up the firewall
-firewall_type="UNKNOWN"		# Firewall type (see /etc/rc.firewall)
+firewall_script="/etc/ipfw.rules" # Which script to run to set up the firewall
+firewall_type="open"		# Firewall type (see /etc/rc.firewall)
```

Comment:
This will enable the IPFW firewall by default with a different rules file, rather than using the rc.firewall setup script for auto-generation. Not sure if you want this on TrueOS....